lib/OpenQA/Schema/Result/Workers.pm

```diff
@@ -78,14 +78,13 @@ __PACKAGE__->inflate_column(
         deflate => sub { encode_json(shift) },
     });
 
-# TODO
-# INSERT INTO workers (id, t_created) VALUES(0, datetime('now'));
-
 sub name {
     my ($self) = @_;
     return $self->host . ":" . $self->instance;
 }
 
+sub to_string { name(@_) }
```

Comment:
This is not a library where we need to keep certain functions for API compatibility. So I'd prefer if you removed the function completely replacing all usages.

Reply:
I agree. I will do that later :)